### PR TITLE
add rp docker file / test

### PR DESCRIPTION
**Pull request by andre-merzky**

### CODE REVIEW

File: docker/rp/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ENV RADICAL_PILOT_DBURL=mongodb://localhost/am
+
+# RP install
+ENV VIRTUAL_ENV=/ve_rp
+RUN python3 -m venv $VIRTUAL_ENV
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+RUN which python3
+RUN pip install --upgrade pip setuptools
+RUN pip install pytest
+RUN pip install radical.pilot
```
Comment (SteVwonder) [inline docker/rp/Dockerfile:30]: Unfortunately with Docker, each `RUN` results in a whole new layer in the dockerfile, and there are typically only 128 layers allowed (apparently it is actually an undocumented amount: https://github.com/docker/docker.github.io/issues/8230).

It is gross, but the "best practice" is to chain all of these together into a single `RUN` to save layers.

**Review by SteVwonder — COMMENTED** [review]
Thanks @andre-merzky!  Few comments:

File: docker/rp/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,37 @@
+
+# RP DOCKERFILE
+
+# The ExaWorks SDK Base
+# this expectes something like `docker build base -t base`
```
[the remark this answers is not in the document]
Comment (andre-merzky) [reply docker/rp/Dockerfile:5]: Thanks!

File: docker/rp/Dockerfile (hunk 1 above)
Comment (andre-merzky) [reply docker/rp/Dockerfile:30]: I really don't have much experience with docker, so very much appreciate this info!  Will change.

Comment (andre-merzky) [reply docker/rp/Dockerfile:30]: BTW, this reminds me of [_"INTERCAL, YAML, And Other Horrible Programming Languages"_](https://blog.earthly.dev/intercal-yaml-and-other-horrible-programming-languages/)...

**Review by SteVwonder — APPROVED** [review]
LGTM! Thanks @andre-merzky

**Pull request merged**